2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs

```diff
@@ -56,11 +56,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IMSGraphService, MSGraphService>();
 
             // Sign-in users with the Microsoft identity platform
-            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                .AddSignIn(options =>
+            services.AddSignIn(options =>
                 {
-                    Configuration.Bind("AzureAD", options);
```

Comment:
I think we still need `Configuration.Bind("AzureAD", options);` both here and in line 88.

Reply:
I've fixed it. thx

---
In reply to: [406889283](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/331#discussion_r406889283) [](ancestors = 406889283)